en/contributing/project-requirements.md

```diff
@@ -1,21 +1,21 @@
 # Project requirements
 
-## Download Git
-We recommend downloading Git from [http://git-scm.com/downloads](http://git-scm.com/downloads)
+## Загрузить Git
+Мы рекомендуем загрузить Git с [http://git-scm.com/downloads](http://git-scm.com/downloads)
```

Comment:
from translate c in russian or is a typo?

Reply:
My bad, I was working with two files and accidentally add translation to the English version :( I will create a new pull request.